### PR TITLE
(2) Updated Models

### PR DESCRIPTION
**Pull request by BenRJG**
Updated models for Rviz sim including code changes for the relevant parameters.
Rejigged a bit of code moving the code linking the arms to the robot from tracebot_mockup_macro.urdf.xacro to tracebot_mockup.urdf.xacro. This keep the macro solely for the robot base, and prevents duplications when calling in the macro for the gazebo sim.

### CODE REVIEW

File: tracebot_mockup_description/urdf/tracebot_mockup_macro.urdf.xacro [hunk 1]
```diff
@@ -1,127 +1,54 @@
...
-      <child link="table"/>
+      <child link="robot_mount_base"/>
     </joint>
 
     <link name="robot_mount_base">
       <visual>
         <origin xyz="0 0 ${robot_mount_height/2}" rpy="0 0 0"/>
         <geometry>
-          <box size="${robot_mount_length} ${robot_mount_width} ${robot_mount_height}"/>
+          <mesh filename="file:///home/benrjg/astech_wss/tracebot_mockup_ws/src/tracebot_mockup/tracebot_mockup_description/meshes/stand.stl" />
```
Comment (miguelprada) [inline tracebot_mockup_description/urdf/tracebot_mockup_macro.urdf.xacro:27]: Should be `package://tracebot_mockup_description/meshes/stand.stl` instead.

**Review by BenRJG — COMMENTED** [review]
Changed location for mesh file

**Review by miguelprada — CHANGES_REQUESTED** [review]
As discussed on the phone, it would be nice if the instancing of the UR arms still happened inside the macro. That way the macro would be completely self-contained and (almost) _hypothetically_ able to be instanced multiple times (some sort of general _prefix_ would be required to make that fully true).

I'll try to look into this in parallel, in case you face problems making this play nice with the _future_ introduction of the gazebo simulation package.

**Comment by BenRJG** [comment]
> As discussed on the phone, it would be nice if the instancing of the UR arms still happened inside the macro. That way the macro would be completely self-contained and (almost) _hypothetically_ able to be instanced multiple times (some sort of general _prefix_ would be required to make that fully true).
> 
> I'll try to look into this in parallel, in case you face problems making this play nice with the _future_ introduction of the gazebo simulation package.

I can take a look at this now if you'd like, with playing around with similar problems doing the camera I think I've figured out a nice way to integrate it

**Pull request closed without merging**